plugins/outputs/sql/README.md

```diff
@@ -155,6 +164,29 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   #  # conversion_style = "unsigned_suffix"
 ```
 
+## Schema updates
+
+The default behavior of this plugin is to create a schema for the table,
+based on the current metric (for both fields and tags). However, writing
+subsequent metrics with additional fields or tags will result in errors.
+
+If you wish the plugin to sync the column-schema for every metric,
+specify the `table_update_template` setting in your config file.
+
+> [!NOTE] The following snippet contains a generic query that your
+> database may (or may not) support. Consult your database's
+> documentation for proper syntax and table / column options.
+
+```toml
+# Save metrics to an SQL Database
+[[outputs.sql]]
+  ## Table update template, available template variables:
+  ##  {TABLE} - table name as a quoted identifier
+  ##  {COLUMNS} - column definitions (list of quoted identifiers and types)
+  ## NOTE: Ensure the user (you're using to write to the database) has necessary permissions
```

Comment:
Maybe simplify it even more, as it already is inconsistent with the sample config:
```suggestion
  ## Table update template
```

Reply:
Thanks, somehow missed this one